src/elements/content-sidebar/additional-tabs/withFtuxTooltip.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow
+ * @file Sidebar Additional Tab FTUX tooltip HOC
+ * @author Box
+ */
+
+import * as React from 'react';
+
+import AdditionalTabFtuxTooltip from './AdditionalTabFtuxTooltip';
+import type { AdditionalSidebarTabFtuxData } from '../flowTypes';
+
+type Props = {
+    ftuxTooltipData?: AdditionalSidebarTabFtuxData,
+    isLoading: boolean,
+};
+
+const withFtuxTooltip = (WrappedComponent: React.ComponentType<any>) => {
```

Comment:
If this is only used once (within AdditionalTab), does it need to be a HOC at all? It seems like the logic is relatively simple and could be inlined in AdditionalTab's render logic.

Reply:
I originally had it in AdditionalTab, but @ConradJChan suggested moving it to a HOC. I'm not really an expert on front-end patterns, but I'm happy to put it wherever you think makes the most sense

Reply:
It looks like it was in HOC form in the initial commit, just as a method rather than a component. Maybe we could modify `AdditionalTabFtuxTooltip `to accept an `isVisible` prop and return `children` directly if it's false?

Reply:
Ah, I see what you're saying. Doing it in AdditionalTabFtuxTooltip would be pretty straightforward. Do you think it would be confusing to have the component no-op based on the isVisible prop? Or have we followed that pattern with other components?

Reply:
It's basically what the HOC is doing, right? We just accomplish it with one fewer component. I think simplicity should win out here, especially since none of this is being reused elsewhere.